### ISSUE TITLE
what is the difference between Generator's learning rate and Discriminator's  learning rate

### ISSUE THREAD

**Issue by 123srikanth** [kan-bayashi/ParallelWaveGAN#317]
Hi there I have used learning rate of 0.00025 in espnet tts training so while changing the same in vocoder I'm confused since there are two learning rates i.e, Generator's learning rate and Discriminator's  learning rate, which one I should change?

**Comment by kan-bayashi** [comment]
The training of text2mel and mel2wav part is totally different, so I do not  recommend using the same parameters.

And it seems that you need to understand GAN-based training and I recommend reading the papers at first.
https://github.com/kan-bayashi/ParallelWaveGAN#references

**Comment by 123srikanth** [comment]
thank you
